### ISSUE TITLE
[Bug]: Incorrect bruker metadata parsing for single plane imaging

### ISSUE THREAD

**Issue by mhturner** [catalystneuro/roiextractors#341]
### What happened?

Using the Bruker->NWB converter in neuroconv (as [here](https://neuroconv.readthedocs.io/en/main/conversion_examples_gallery/imaging/brukertiff.html#)), I noticed that my single-plane Bruker imaging data was being flagged as multiplane.

Tracked this down to `brukertiffimagingextractor._determine_imaging_is_volumetric()`, which assumes that the metadata flag 'zDevice' == 1 means this is a volumetric scan, which is a sensible thing to assume. However my single plane image data as well as my multiplane image data both have this value as 1.

In my experience, a reliable way to tell what the scan type is from the .xml metadata is the Sequence type. This can be things like:
'TSeries Timed Element' - XYT image data
'TSeries ZSeries Element' - XYZT image data
'ZSeries' - XYZ data (not a time series)
'Single' - XY image

and probably other things too that I don't acquire.

### Steps to Reproduce

```python
from roiextractors.extractors.tiffimagingextractors.brukertiffimagingextractor import _determine_imaging_is_volumetric

print(_determine_imaging_is_volumetric(bruker_file_dir))
```

### Traceback

```python
True
```

### Operating System

macOS

### Python Executable

Conda

### Python Version

3.10

### Code of Conduct

Yes

### Duplicated Issue Check

No

**Comment by CodyCBakerPhD** [comment]
Thanks for finding this and including all the details!

attn @weiglszonja @alessandratrapani @pauladkisson

**Comment by weiglszonja** [comment]
Thank you for opening this issue @mhturner! I'm thinking that since it is not obvious to decide whether it is a volumetric type, how about we just don't assume and separate the classes to single and multiplane, and then the user decides what is the more appropriate extractor/interface for the data. I'm thinking a refactor similar to ScanImage. What do you guys think? @pauladkisson @alessandratrapani @CodyCBakerPhD

**Comment by h-mayorquin** [comment]
@mhturner would you be so kind to share the corresponding xml of your data? 

The volumetric data that I have seen has a specification that looks like this:

```xml
      <PVStateShard>
        <PVStateValue key="framePeriod" value="0.048474237" />
        <PVStateValue key="laserPower">
          <IndexedValue index="0" value="650.390625" description="920 Axon" />
          <IndexedValue index="1" value="0" description="1064 Axon" />
          <IndexedValue index="2" value="0" description="Monaco" />
        </PVStateValue>
        <PVStateValue key="positionCurrent">
          <SubindexedValues index="XAxis">
            <SubindexedValue subindex="0" value="14.927" />
          </SubindexedValues>
          <SubindexedValues index="YAxis">
            <SubindexedValue subindex="0" value="56.215" />
          </SubindexedValues>
          <SubindexedValues index="ZAxis">
            <SubindexedValue subindex="0" value="166.575" description="Z Focus" />
            <SubindexedValue subindex="1" value="130" description="Optotune ETL 10-30" />
          </SubindexedValues>
```

The volumetric example that we have in the gin data is also like this.

**Comment by mhturner** [comment]
Here is a corresponding part of a metadata file. This is from an XYT scan. Let me know if you would like me to send you this file (can't attach .xml here).

<img width="1176" alt="Screenshot 2024-06-07 at 4 12 59 PM" src="https://github.com/catalystneuro/roiextractors/assets/9029384/2b417aaa-b872-4db1-870b-51fb4bb98716">

**Comment by h-mayorquin** [comment]
Yes, can you send me the file? one option is my email, it is just my user name here in github but with a dot instead of an underscore. Gmail.

**Comment by h-mayorquin** [comment]
Thanks a lot @mhturner. I double checked my files and while the xml do have the information that I listed here I think your solution has the virtue of being way way simpler. So as long as we don't have the documentation of Praire view's xmls I think we will have to run with this heuristics and let's start with yours.

**Comment by h-mayorquin** [comment]
Hey @mhturner, any chance you have an xml corresponding to a a `TSeries ZSeries Element` with two channels?

**Comment by mhturner** [comment]
Sure - here is a dropbox link with such a file: https://www.dropbox.com/scl/fi/ka2j71jakd9xc0vje4ys1/TSeries-20231114-001.xml?rlkey=rm7885ktloo9qiri7l4nc6rr3&dl=0

**Comment by h-mayorquin** [comment]
Thanks a bunch for the second file @mhturner . I implemented your suggestion and now is on main. I will keep this open till we solve your problems.
